src/controller_model/distributionList.ts

```diff
@@ -22,18 +22,28 @@ import {WebClientService} from '../services/webclient';
 // Type aliases
 import ControllerModelMode = threema.ControllerModelMode;
 
-export class DistributionListControllerModel implements threema.ControllerModel<threema.DistributionListReceiver> {
+export class DistributionListControllerModel
+        implements threema.ControllerModel<threema.DistributionListReceiver>,
+                   threema.ControllerModelWithMembers {
+    // Angular services
     private $translate: ng.translate.ITranslateService;
     private $mdDialog: ng.material.IDialogService;
+
+    // Custom services
     private readonly log: Logger;
-    public members: string[];
-    public name: string;
+    private readonly webClientService: WebClientService;
+
+    // Fields required by interface
+    public readonly receiverType = 'contact';
```

Comment:
Oh nice. How did this not break something when testing it?!

One of the nice side effects of trying to add more structure, it increases the surface for bugs.

Reply:
No failure because unused (messenger uses this for mentions, which are only allowed in groups).